$OpenBSD: patch-wo_pcgi_py,v 1.3 2003/05/14 21:35:52 krw Exp $

        We don't need to build Data.fs, start/stop, or inituser;
        these are handled by zope-instance.

--- wo_pcgi.py.orig	Wed Aug 14 17:11:39 2002
+++ wo_pcgi.py	Wed Apr 16 20:33:59 2003
@@ -10,6 +10,14 @@
 # FOR A PARTICULAR PURPOSE
 #
 ##############################################################################
+
+# NOTICE AS REQUIRED BY LICENSE
+#
+# August 5, 2001
+#
+# Removed building of Data.fs, start/stop, and inituser; these are handled
+# by our zope-instance script.
+
 """Try to do all of the installation steps.
 
 This must be run from the top-level directory of the installation.
@@ -31,12 +39,6 @@
     walkandscrub.walkandscrub(home)
     import compilezpy
     import build_extensions
-    user=group=''
-    import default_content; default_content.main(home, user, group)
-
-    pcgi=os.path.join(home, 'Zope.cgi')
-    import make_start; make_start.sh(home, user, group)
-    import zpasswd; zpasswd.write_inituser(home, user, group)
 
     print '-'*78
     print
